oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoDocumentStore.java

```diff
@@ -383,16 +393,17 @@ private MongoDBConnection getOrCreateClusterNodesConnection(@NotNull MongoDocume
         return mc;
     }
 
-    private void ensureIndexes(@NotNull MongoStatus mongoStatus) {
+    private void ensureIndexes(MongoDatabase db, @NotNull MongoStatus mongoStatus) {
```

Comment:
Please add `NotNull` annotation to `db` parameter.

Reply:
Addressed it.